ceph/ceph_admin/dashboard.py

```diff
@@ -112,3 +133,121 @@ def validate_enable_dashboard(cls, username, password):
         LOG.info(
             "Dashboard API login is successful, status code : %s" % resp.status_code
         )
+
+
+def enable_alertmanager(cls, config):
```

Comment:
Could we write a separate class for `alertmanager`? You could call the instance of alertmanager in this module. This way we can expand/extend `alertmanager` and be modular.

Reply:
Let's pick this up in another PR